apps/throttles.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework.throttling import SimpleRateThrottle
+from influxdb_metrics.loader import log_metric
+
+
+class OrgApiLimit(SimpleRateThrottle):
+    """
+    API rate limiter for req/day
+    """
+    scope = 'org_api_limit'
+    rate = '1000/day'  # default rate
+
+    def get_cache_key(self, request, view):
+        # Verify user belongs to billable organization
+        if request.user.is_authenticated and 'payload' in request.user.token \
```

Comment:
Where does 'payload' come from? As far as I know you should be able to get the organization ID and api_limit directly from the token.

Reply:
Also check out the get_user() helper in permissions.py. This might help you simplify some of your logic in this `if` statement

Reply:
This makes much more sense. When checking out JWT for the first time I inspected the JWT token and saw an org ID in 'payload' so thought that was the target. Looks like something like request.user.token.get('organization_id', None) is the way to go.